### PR TITLE
release 6.3.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,11 +27,17 @@ These should not affect the functionality of the module.
...
 **Merged pull requests:**
 
+- release 6.2.0 [\#388](https://github.com/voxpupuli/puppet-r10k/pull/388) ([bastelfreak](https://github.com/bastelfreak))
 - Add logging messages [\#385](https://github.com/voxpupuli/puppet-r10k/pull/385) ([amateo](https://github.com/amateo))
 
 ## [v6.1.0](https://github.com/voxpupuli/puppet-r10k/tree/v6.1.0) (2017-06-24)
+
 [Full Changelog](https://github.com/voxpupuli/puppet-r10k/compare/v6.0.0...v6.1.0)
 
+**Breaking changes:**
```
Comment (alexjfisher) [inline CHANGELOG.md]: ???

Comment (bastelfreak) [reply CHANGELOG.md]: latest changelog generator now supports a section for stuff tagges with backwards-incompatible

Comment (alexjfisher) [reply CHANGELOG.md]: But releasing 6.0.0 is not a breaking change of 6.1.0

Comment (bastelfreak) [reply CHANGELOG.md]: you added that label to the corresponding PR :P I removed it and regenerated the changelog

Comment (alexjfisher) [reply CHANGELOG.md]: Do I didn’t! Why am I getting the blame for everything today! :)

**Pull request merged**